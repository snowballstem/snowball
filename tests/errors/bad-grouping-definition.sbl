groupings (g1 g2 g3)
define g1
define g2 'abc' -
define g3
externals (stem)
define stem as ()
