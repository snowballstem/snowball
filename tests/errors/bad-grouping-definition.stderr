errors/bad-grouping-definition.sbl:3: unexpected define in grouping definition after name
errors/bad-grouping-definition.sbl:4: unexpected define in grouping definition after -
errors/bad-grouping-definition.sbl:5: unexpected externals in grouping definition after name
